### ISSUE TITLE
Precompilation failure on Julia nightly

### ISSUE THREAD

**Issue by bkamins** [JuliaData/Missings.jl#134]
I get the following warning:
```
WARNING: Method definition (::Missings.PassMissing{F})(Any...) where {F} in module Missings at /home/bkamins/.julia/packages/Missings/sx5js/src/Missings.jl:192 overwritten on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition (::Missings.PassMissing{F})(Any) where {F} in module Missings at /home/bkamins/.julia/packages/Missings/sx5js/src/Missings.jl:184 overwritten on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **
```
on current Julia nightly ( 1.8.0-DEV.31 (2021-06-14), Commit 2f1a958826

@nalimilan - do you know what could cause this?

**Comment by nalimilan** [comment]
Maybe a Julia bug with generated functions? Probably worth reporting.

**Comment by bkamins** [comment]
I have the same in https://github.com/KristofferC/Crayons.jl/issues/45. Who do you think we should report to?

**Comment by bkamins** [comment]
Fixed on 1.8.0-DEV.56 (2021-06-19)
